FreeRTOS/FreeRTOS
# Update runtime library support to automatic

Description
-----------
This ensures that CCS builds this library when the project is first built after CCS installation.

Test Steps
-----------
Build CORTEX_R4_RM48_TMS570_CCS5 demo.

Checklist:
----------


- [NA] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS/pull/1273


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
